superset-frontend/src/dashboard/components/BuilderComponentPane.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-env browser */
+import React from 'react';
+import Tabs from 'src/common/components/Tabs';
+import { StickyContainer, Sticky } from 'react-sticky';
+import { ParentSize } from '@vx/responsive';
```

Comment:
I'm surprised to see VX make an appearance here... but looking at this, it looks like this module might solve another problem I have, so this kind of makes my day!